src/aof.c

```diff
@@ -1725,6 +1725,9 @@ int loadSingleAppendOnlyFile(char *filename) {
     server.current_client = old_cur_client;
     server.executing_client = old_exec_client;
     fclose(fp);
+    /* Reclaim page cache memory used by the AOF file in background. */
+    int fd = open(aof_filepath, O_RDONLY);
+    if (fd >= 0) bioCreateCloseJob(fd, 0, 1);
```

Comment:
what about `reclaimFilePageCache(fileno(fp))` before `fclose` to avoid reopen it.

Reply:
if so maybe we can dup the file fd, and close it in the bio.

Reply:
sounds good